source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -272,3 +272,21 @@ export function addCancelSelectActionEventHandler(element: HTMLElement) {
         e.cancelBubble = true;
     };
 }
+
+export function SetDateOrTimePickerVisibility(input: HTMLInputElement, value: string | undefined, show: boolean) {
+    if (!!show) {
+        // shows the time/date picker icon
+        input.readOnly = false;
```

Comment:
Why this?

Reply:
to update visibility of date and time pickers